### DISCUSSION TITLE
One entry for many urls

### DISCUSSION THREAD

**Discussion by gianlucavagnuzzi** [keepassxreboot/keepassxc#11069] [Q&A] [answered]
There is a way to use one entry for many urls!

I've a password that match all 192.168.111 subnet host.
I've try to insert url as http://192.168.111.* or //http://192.168.111.*// or windows title in auto-fill but nothing works!

Thanks in advance

**Comment by droidmonkey** [comment]
What are you trying to accomplish?

**Reply by gianlucavagnuzzi** [reply]
I'm trying to use a KeepassXC record for many url.
I've created an entry as:
![image](https://github.com/user-attachments/assets/01fe1c37-1dd3-4eba-8676-0b00781c8c6f)

but in Browser it doesn't autocompile:
![image](https://github.com/user-attachments/assets/70a02b48-02d5-4f33-9d39-6bec819acc0a)

**Reply by droidmonkey** [reply]
We don't support wild cards, yet. You'll have to add each IP address you want to match to. You can add more urls in the browser tab on the left side when editing an entry.

**Reply by gianlucavagnuzzi** [reply]
Thanks @droidmonkey !
are there any other workarounds? regex?

**Reply by droidmonkey** [reply, marked as answer]
Not until this is merged: https://github.com/keepassxreboot/keepassxc/pull/9835

**Reply by gianlucavagnuzzi** [reply]
Thanks @droidmonkey !
